### DISCUSSION TITLE
The problem after converting the code to exe

### DISCUSSION THREAD

**Discussion by Wizya** [kaliiiiiiiiii/Selenium-Driverless#230] [Q&A]
The code works fine, but if I convert it to an exe file using pyinstaller, I get an error

import time
from selenium_driverless.sync import webdriver
options = webdriver.ChromeOptions()
driver = webdriver.Chrome(options=options)
time.sleep(1000)

![image](https://github.com/kaliiiiiiiiii/Selenium-Driverless/assets/62518688/c01f4584-9df5-4306-a814-0eae2744153c)

**Comment by ttlns** [comment]
has already been answered in issues

**Reply by Wizya** [reply]
Can I get a link to this?
